riscv-c-api.md

```diff
@@ -331,6 +345,8 @@ int bar() {
 }
 ```
 
+The `priority` accepts a digit as the version priority during [Version Selection](#version-selection). If `priority` isn't specified, then the priority of version defaults to zero.
```

Comment:
Now that we accept -1, we have to state ordering.  Does "-1" compare less than "1" (i.e. signed ordering), or does it mean "maximum priority" (i.e. unsigned ordering)?  I think we probably want unsigned here?

Reply:
I assume the signed ordering here, and I also use it in the LLVM implementation. It's just more intuitive to me, but I can change to unsigned ordering if necessary.

Reply:
Why do we need to allow signed numbers at all?

Reply:
@preames do you see any reason to support negative numbers here? The number of target_clones/target_version attributes should be small enough that setting even a smallish number like 1000 should be high enough to be maximum priority.

Reply:
Nope, restricting it to positive only would be fine.  I just want to make sure the interpretation is unambiguous.

Reply:
Removing negative numbers from the priority appears to be a clearer solution to avoid ambiguity. I will update this in the pull request.